### ISSUE TITLE
Foundry V12: Replace deprecated functions.

### ISSUE THREAD

**Issue by farling42** [farling42/fvtt-pdf-pager#89]
Use of `game.template.Item.types` and `game.template.Item.types` is deprecated in Foundry V12.

The warning in the console is:
```
Error: Game#template is deprecated and will be removed in Version 14. Use cases for Game#template should be refactored to instead use System#documentTypes or Game#model
Deprecated since Version 12
Backwards-compatible support will be removed in Version 14
    at Object.logCompatibilityWarning (foundry-esm.js:4488:19)
    at get template (foundry.js:10290:19)
    at updateSheets (pdf-actorsheet.mjs:217:29)
    at configureActorSettings (pdf-actorsheet.mjs:257:3)
    at Object.fn (pdf-config.mjs:310:2)
    at #call (foundry.js:733:20)
    at Hooks.callAll (foundry.js:690:17)
    at Game.setupGame (foundry.js:8972:11)
    at async Game._initializeGameView (foundry.js:10257:5)
    at async Game.initialize (foundry.js:8879:5)
```

**Comment by farling42** [comment]
`game.model.Actor` - includes `base` in the Simple World Building.
`game.system.documentTypes.Actor` - does not include `base` in the Simple World Building system.

So we should `game.system.documentTypes` instead of `game.template`

**Comment by farling42** [comment]
on Foundry V11, `game.system.documentTypes.Actor` is an array:
![image](https://github.com/farling42/fvtt-pdf-pager/assets/17485384/127e46cd-5902-4b0d-8019-1c2436c660fa)

on Foundry V12, `game.system.documentTypes.Actor` is an object:
![image](https://github.com/farling42/fvtt-pdf-pager/assets/17485384/3e7b649d-e5ac-42b3-ba26-61efa139af80)

**Comment by farling42** [comment]
`game.model.actor`

Foundry V11:
![image](https://github.com/farling42/fvtt-pdf-pager/assets/17485384/cf3f79bf-5aad-42b3-ad85-9587757c5027)

Foundry V12:
![image](https://github.com/farling42/fvtt-pdf-pager/assets/17485384/b86fef67-e283-4f16-bb8c-8516aae078a0)

**Comment by farling42** [comment]
`Actor.createDialog` gets a list of the character types:

V11: 
```
      const documentName = this.metadata.name;  // Actor
      const types = game.documentTypes[documentName].filter(t => t !== CONST.BASE_DOCUMENT_TYPE);
```

V12:
```
      const documentName = this.metadata.name;
      const types = game.documentTypes[documentName].filter(t => t !== CONST.BASE_DOCUMENT_TYPE);
```

**Comment by farling42** [comment]
Implemented in 0.52.3
